src/transformers/models/led/modeling_led.py

```diff
@@ -411,29 +411,26 @@ def _chunk(hidden_states, window_overlap, onnx_export=False):
             return hidden_states.as_strided(size=chunk_size, stride=chunk_stride)
 
         # When exporting to ONNX, use this separate logic
```

Comment:
Wondering if it would make sense to include this in an `else` statement to make the code a bit clearer to read?

Reply:
Personally agree it's more readable. I will leave it as is following https://github.com/huggingface/transformers/pull/17176#discussion_r915711144